avocado_vt/plugins/vt_joblock.py

```diff
@@ -111,14 +111,21 @@ def _lock(self, job):
                     except OSError:
                         self.log.warn("Unable to remove stale lock: %s", path)
 
+    @staticmethod
+    def _get_klass_or_none(test_factory):
```

Comment:
I think `_get_klass` is enough for the naming, but that is just a suggestion, code looks good to me.